client/src/pages/Home.tsx

```diff
@@ -143,19 +146,32 @@ function Home(this: any) {
 
   // Testing function to call Sentiment analysis fn...
   const getSentiment = () => {
-    const body = JSON.stringify({ text: inputValue });
-    fetch("/api/sentiment/process", { ...postRequestOptions, body })
-      .then((response) => response.json())
-      .then((data) => {
-        console.log(data);
-        setSentimentText(data.sentiment);
-        setSentimentScore(Math.round(data.score * 100));
-        setEmotionLabel(data.emotions.toString());
-      })
-      .catch((e) => {
-        // Log this error instead of showing on screen
-        console.log(`Call to /api/sentiment/process failed. Error: ${e}`);
-      });
+    if (inputValue.trim() === "") {
+      // When the input is empty
+      // set all the states if a user clicks on the sentiment button
+      // this should also hold true and follow thru too (the states which are set)
+      // when a user inputs something in the text box and clicks on it
+      setSentimentText("None");
+      setSentimentScore(0);
+      setEmotionLabel("Not set");
+      setSentimentPlaceholder("Sentiment analysis of the text is:");
+      setEmotionalPlaceholder("Emotional analysis result:");
+      setPercent("%");
```

Comment:
A lot of states here, not sure about the others but if `percent` is just the `%` symbol then it doesn't need its own state lol.

Reply:
yeah I couldn't really think of anything else to get it to work, i'm sure there must be better ways to do it but yeah we can def look into refactoring it